types/rdf-loader-code/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for rdf-loader-code 0.3
+// Project: https://github.com/zazuko/rdf-loader-code
+// Definitions by: tpluscode <https://github.com/tpluscode>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.4
+
+export {};
```

Comment:
heh, this threw me off, but there isn't a file for this at all in the JS 👍🏻

Reply:
Yup, but as I understand this is how it's exported for DT consumption (and there is no warnings in linter for that :D )